xds/src/main/java/io/grpc/xds/internal/sds/ServerSslContextProviderFactory.java

```diff
@@ -40,7 +40,11 @@
 
   ServerSslContextProviderFactory(
       Bootstrapper bootstrapper, CertProviderServerSslContextProvider.Factory factory) {
-    this.bootstrapper = bootstrapper;
+    try {
+      this.bootstrapInfo = bootstrapper.bootstrap();
```

Comment:
Read bootstrap in constructor? I don't know exactly how the whole workflow of the bunch of providers/factories. But this doesn't seem to be right. Also, throwing runtime exceptions in such an ad-hoc way looks very bad.

Reply:
refactored to move the bootstrap file reading to the `TlsContextManagerImpl`